src/strategies/math/options.ts

```diff
@@ -3,6 +3,7 @@ export type Operand = StrategyOperand | ConstantOperand;
 export interface Options {
   operands: Operand[];
   operation: Operation;
+  multiplier: number;
```

Comment:
Hi @Skyge It should be like other types, you can pass multiplier into operands like other operations

Reply:
Okay, let me update it.

Reply:
So do you mean like this:
```
"operands": [
          {
            "type": "strategy",
            "strategy": {
              "network": "1",
              "name": "erc20-balance-of",
              "params": {
                "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
                "symbol": "DAI",
                "decimals": 18
              }
            },
            "multiplier": 6
          }
        ],
```